README.md

```diff
@@ -397,6 +399,15 @@ NODE_ENV=test node app.js
 
 ### Running Tests
 
+Starting from version `1.6.0`, Lisk will be using [Jest](https://jestjs.io) as its main test runner and gradually deprecate [mocha](https://mochajs.org). Since, rewriting all existing mocha tests is not feasible at the moment, we are going to have 2 test runners in our code base.
```

Comment:
```suggestion
Starting from version `1.6.0`, Lisk Core will be using [Jest](https://jestjs.io) as its main test runner and gradually deprecate [mocha](https://mochajs.org). Since rewriting all existing mocha tests is not feasible at the moment, we are going to have two test runners in our code base:
```